test/datasets_utils.py

```diff
@@ -561,9 +561,9 @@ def test_feature_types(self, config):
     @test_all_configs
     def test_num_examples(self, config):
         with self.create_dataset(config) as (dataset, info):
-            assert len(dataset) == info["num_examples"]
+            assert len(dataset) == info["num_examples"], f"The number of examples {len(dataset)} does not match the expected {info['num_examples']}"
 
-    @test_all_configs
+    @ test_all_configs
```

Comment:
Looks like there's a few formatting issues like these ones. We try to avoid formatting changes in PRs like these, because they can add noise to `git blame`

As you can see there are linting issues: https://app.circleci.com/pipelines/github/pytorch/vision/18822/workflows/40d854f9-fd3b-43cb-9471-9eb90d4c93ac/jobs/1522929

We have instructions for applying formatters here: https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#formatting